docker/comfyui-cuda12.8.dockerfile

```diff
@@ -9,8 +12,10 @@ RUN apt update -y && apt install -y \
     python3 \
     python3-pip \
     python3-venv \
-    unzip \
-    && rm -rf /var/lib/apt/lists/*
+    unzip
+
+# Clean up to reduce image size
+RUN apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/*
```

Comment:
As a separate `RUN`, the intermediate files will still exist in the preceding layer. This will not reduce the image size as expected. These should be chained in the preceding command.

What is being removed by `apt-get autoremove`?